docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "K8ssandra site engineer quick start"
+linkTitle: "Site engineers"
+weight: 2
+description: |
+  Take a tour of your K8ssandra cluster using all the utilities!
+---
+
+**Completion time**: **10 minutes**.
+
+* [Upgrading K8ssandra with Ingress support]({{< relref "#configure-ingress" >}}): Enabling access to K8ssandra from outside the K8s cluster via Traefik.
+* [Accessing K8ssandra utilities]({{< relref "#access-k8ssandra-utilities" >}}): Accessing useful utilities like the Cassandra Reaper repair tool and Grafana metrics reporting.
+* [Starting and stopping K8ssandra]({{< relref "#cassandra-operations" >}}): Cleanly stopping and restarting the K8ssandra pod.
+
+## Configure Ingress
+
+Right now, you can only interact with your Cassandra node within the K8s cluster using `kubectl` commands which is a pretty severe limitation. To enable external access for applications as well as enable access to K8ssandra features like Grafana dashboards, the Reaper Repair UI, and the Stargate GraphQL playground, you'll need to configure Ingress. In this section we'll upgrade your existing K8ssandra installation to support Ingress using a Traefik Helm chart.
+
+{{% alert title="Tip" color="success" %}}
+Kubernetes Ingress configuration is a complicated topic. Really all that is happening in this upgrade is that a set of external URIs and ports are being mapped to internal Kubernetes resources. It's not necessary to understand all of the configuration particulars in order to successfully complete this section.
+{{% /alert %}}
+
+To upgrade K8ssandra:
+
+1. Copy the following YAML into a new file named `k8ssandra-traefik.values.yaml`:
+
+    ```yaml
+    cassandra:
+      version: "3.11.7"
+      auth:
+        enabled: false
+      clusterName: k8ssandra
+      datacenters:
+      - name: dc1
+        size: 1
+    ingress:
+      traefik:
+        enabled: true
+        repair:
+          enabled: true
+          host: repair.127.0.0.1.nip.io
+        stargate:
+          enabled: true
+          host: stargate.127.0.0.1.nip.io
+    kube-prometheus-stack:
+      prometheus:
+        enabled: true
+        prometheusSpec:
+          externalUrl: http://localhost:9090/prometheus
+          routePrefix: /prometheus
+        ingress:
+          enabled: true
+          paths:
+            - /prometheus
+      grafana:
+        enabled: true
+        ingress:
+          enabled: true
+          path: /grafana
+        adminUser: admin
+        adminPassword: admin123
+        grafana.ini:
+          server:
+            root_url: http://localhost:3000/grafana
+           serve_from_sub_path: true
+    ```
+
+1. Copy the following YAML into a file named `traefik.values.yaml`:
```

Comment:
the order of steps is potentially confusing. when I first read it, the filenames are quite similar so I wondered if there was a copy paste error. I'd suggest creating the traefik values file, installing traefik, then creating the k8ssandra traefik values file and applying that.

Reply:
I like it! Why didn't I think of that!